packages/ui/src/router/PopupRouter.tsx

```diff
@@ -109,7 +110,7 @@ const PopupRouter = ({
     const state = useBlankState()!
     const isOnboarded = state?.isOnboarded
     const resetHandler = async () => {
-        await lockApp()
+        chrome.runtime.reload()
```

Comment:
Are we able to access the `chrome.runtime` from the UI? I am not sure about that.

Reply:
Seems to work 👍

Reply:
💪